README.md

```diff
@@ -5,6 +5,26 @@ Generic encryption software
 
 ## Usage
 
+encryptio uses a CLI interface(using [argparse](https://docs.python.org/3/library/argparse.html)): 
+
+```bash
+python3 encryptio.py [-h] encryptionAlgorithm encryptionKey encryptionLibrary encryptionObjects (encryptionObjectOptions)
```

Comment:
sollen wir da zwischen linux und windows unterscheiden?

Reply:
There is no difference between Linux and Windows when calling the script, you can execute the python script on windows and linux with 'python3 encryptio.py ...'